source/type/data.cpp

```diff
@@ -484,6 +489,26 @@ void Data::aggregate_odt() {
     }
 }
 
+void Data::fill_stop_point_address() {
+    size_t without_address = 0;
```

Comment:
_unsigned_ is better ? like **uint32** to avoid overflow?

Reply:
std::size_t is already unsigned I think. (could be u32 or u64 depending on the platform)